setup.cfg

```diff
@@ -26,3 +26,9 @@ disallow_untyped_defs = True
 ignore_missing_imports = True
 strict_optional = True
 warn_no_return = True
+
+[semantic_release]
+version_variable = "amundsenfrontendlibrary/setup.py:__version__",
```

Comment:
nice, qq: how does the version get updated?

Reply:
the semantic-release library would do it for us.